modules/appeals_api/app/services/appeals_api/pdf_construction/notice_of_disagreement/v2/pages/long_data_and_extra_issues.rb

```diff
@@ -34,7 +34,7 @@ def build!
           attr_accessor :pdf, :form_data
 
           def no_content
-            !extra_issues? && !form_data.long_preferred_email? && form_data.long_rep_name?
+            !extra_issues? && !form_data.long_preferred_email? && !form_data.long_rep_name?
```

Comment:
hard to spot, but form_data.long_rep_name? was missing the not operator.  This was causing no_content to return false(ie there IS extra content) when in fact there was no extra content and no need for an extra overflow page

Reply:
Nice catch!